### PR TITLE
New version: ShiningLight.OpenSSL version 3.0.5

### LINKED ISSUE #65414: [Package Request]: Update OpenSSL to latest version
### Package Requested

- [ ] I would like someone else to build the manifest.
- [ ] I would like help so I can submit the manifest.
- [ ] I have performed a search and couldn't find this package.
- [ ] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [X] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Shining Light Productions
Publisher Url: https://slproweb.com/
Package Name: OpenSSL [ShiningLight.OpenSSL]
Description: The Win32/Win64 OpenSSL Installation Project is dedicated to providing a simple installation of OpenSSL for Microsoft Windows.
Package Version: 3.0.5
Installer URL: https://slproweb.com/download/Win64OpenSSL-3_0_5.msi

Current Package Identifier (for new versions): https://slproweb.com/download/Win64OpenSSL-3_0_3.msi

### PR DESCRIPTION
**Pull request by NSO73**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

* Resolves [[Package Request]: Update OpenSSL to latest version #65414](https://github.com/microsoft/winget-pkgs/issues/65414)

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65449)

### CODE REVIEW

**Review by fluentmoheshwar — COMMENTED** [review]
Looks good but I didn't tested it.

File: manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

File: manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: ShiningLight.OpenSSL
+PackageVersion: 3.0.5
```
Comment (Trenly) [inline manifests/s/ShiningLight/OpenSSL/3.0.5/ShiningLight.OpenSSL.installer.yaml:4]:
```suggestion
PackageVersion: 1.0.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]
The PackageVersion must match control panel; When updating this, you will need to update the name of the version folder also

**Review by ItzLevvie — APPROVED** [review]

**Comment by maxsharabayko** [comment]
I've got a question regarding the OpenSSL version in `wget`, but feel hesitant to file a new issue.

If I run `wget search openssl` I get the following:

```shell
> winget search openssl
Name          Id                         Version Match        Source
--------------------------------------------------------------------
OpenSSL       ShiningLight.OpenSSL       1.0.0                winget
OpenSSL Light ShiningLight.OpenSSL.Light 1.0.0   Tag: openssl winget
```

Here I see the version is 1.0.0.
If I run `wget show openssl` I get:

```shell
> winget show openssl
Found OpenSSL [ShiningLight.OpenSSL]
Version: 1.0.0
Publisher: Shining Light Productions
Publisher Url: https://slproweb.com/
Publisher Support Url: https://slproweb.com/support.html
Moniker: openssl
Description: The Win32/Win64 OpenSSL Installation Project is dedicated to providing a simple installation of OpenSSL for Microsoft Windows.
Homepage: https://slproweb.com/products/Win32OpenSSL.html
License: Apache-2.0
License Url: https://www.apache.org/licenses/LICENSE-2.0
Copyright: Copyright (c) Shining Light Productions
Installer:
  Type: wix
  Download Url: https://slproweb.com/download/Win64OpenSSL-3_0_5.msi
  SHA256: 505e1a16e36edc927c889e8acfd677776e3b257c50ff19ac2ee0dfafba2cb0a2
```

It is now obvious the OpenSSL version is 3.0.5. But version 1.0.0 is misleading.
What does "1.0.0" stand for? Is it some manifest version?

**Comment by andrew-flatters-bliss** [comment]
Semi-related to @maxsharabayko 's  point but each time I do a `winget upgrade --all` the version 1.0.0 is showing as installed multiple times but its showing 3.0.7 when I `winget show openssl`

image

**Pull request merged**